lib/graphql/pagination/relation_connection.rb

```diff
@@ -116,9 +116,9 @@ def calculate_sliced_nodes_parameters
         if defined?(@sliced_nodes_limit)
           return
         else
+          next_offset = relation_offset(items) || 0
           if after_offset
-            previous_offset = relation_offset(items) || 0
```

Comment:
Oops, the previous offset was only applied _when_ `after:` was present 😖 . Now it's _always_ considered in the new offset value.